### DISCUSSION TITLE
Welcome to PersistentWindows Discussions!

### DISCUSSION THREAD

**Discussion by kangyu-california** [kangyu-california/PersistentWindows#66] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hdprene** [comment]
Hi I used pw5.11 yesterday to overcome silly app replace issues while using the Speaka 4x1 KMV switch with USB. Got it working on two of the four apps which are handy in my situation to have them opened. The two which still get replaced while I switch are old apps but still working; it's studio clock and time tool which checks the official atom time in Germany with an usb receiver every minute. Are there (secret) additional options to get them persistent too? Thank you for building this app, also for reading and hopefully answering ;-)

**Comment by kangyu-california** [comment]
@hdprene 
Please try release 5.12, in which garyd9 (partially) solved a similar issue #56 manifested by winamp app due to non-regular window hierarchy.
if the issue is not resolved, please provide link(s) to the app such as studio clock, and we will take a look.
There is no guarantee this can be fixed though.

**Comment by hdprene** [comment]
Hi Kangyu!

Thank you very much for answering and help! It encourages me to go on with the switch. Of course there is no guarantee but it feels great to receive assistance from you 😊



Best Regards,
René

**Comment by hdprene** [comment]
Tested the 5.12 and I see no differences with 5.11 The two apps which already listened to PW still do. They both have an internal auto startup with windows option. The other two dont have that. They are started using the user/roaming/windows etc option. On most machines they open horizontal ok but vertical 4 to 4.4 cm higher than the original position (see screen dump)  Every app is started as admin, is that correct?  
[PW.zip](https://github.com/kangyu-california/PersistentWindows/files/5826292/PW.zip)

I will upload the links too. But again, thanks for your help.



![2021-01-17_15h55_41](https://user-images.githubusercontent.com/52597393/104846790-9a514980-58dc-11eb-9441-e282ca9f5b90.jpg)

**Reply by kangyu-california** [reply]
@hdprene 
checkout my comment in
https://github.com/kangyu-california/PersistentWindows/issues/75

**Comment by kangyu-california** [comment]
@hdprene 
Created issue #75, the issue is actually a enhancement request for PW to track and restore child windows such as dialog box.

**Comment by hdprene** [comment]
Thank you! Will test it today

**Reply by hdprene** [reply]
Thank you very very very much this update solved my problems with the Speaka 4 x hdmi KMV switch.  The manufacturer didn't even react on my question but you solved it with persistant windows 5.16!! Great! https://www.conrad.nl/p/4-poorten-kvm-switch-hdmi-usb-speaka-professional-1000168

**Comment by Gangstrocity** [comment]
First I just want to say this software is great, so thanks for making it available for free.

I'm having a small issue when I use remote desktop. If I minimize remote desktop, when I bring it back up it immediately minimizes again and I have to click it again to bring it up. It always stays up the second time. Overall it's not a big deal, just a minor annoyance. I've seen other windows randomly minimize, but remote desktop does it every time.

**Reply by kangyu-california** [reply]
The only scenario this would happen is during PW restore, which could take as much as 20 seconds due to slow Remote Desktop connection, any user move/resize/max/min operation during that critical period would be "reverted".

Readme document suggested to always display PW icon, so that user could see the red sign (indicating PW is busy restoring windows) and avoid any window operation.

If this is not the case, please post the PW event 9990/9999 from event viewer as suggested in Readme.

**Comment by Gangstrocity** [comment]
Thanks for the reply. The icon is not red when it happens. Found this in the event viewer, let me know if you need more information.

PersistentWindows: top left of Rect (-1920, 0), 5360 x 2210 is off-screen
PersistentWindows: fix invisible window "whitlow-doc - Remote Desktop Connection"

**Reply by kangyu-california** [reply]
Release 5.20 resolved an issue related to multi-monitor remote desktop session, please try

**Reply by Gangstrocity** [reply]
This appears to have resolved my issue. Thanks so much.

**Comment by hdprene** [comment]
thank you very much!

**Comment by TheMuffinlord** [comment]
Would there be any possibility of adding the same sort of functionality for the taskbar? I have my taskbar on a secondary monitor for gaming reasons but every time I toggle to my work laptop and back the bar ends up on my primary.

**Reply by kangyu-california** [reply]
PW supported taskbar restore long time back, but only for the one on main display. 
Just make sure to leave your mouse still during restore.

**Reply by kangyu-california** [reply]
Also make sure to unlock the taskbar in taskbar settings,

![image](https://user-images.githubusercontent.com/59128756/116501354-6a173080-a865-11eb-9e4d-8ff0005abe6d.png)
